tikv/split_region.go

```diff
@@ -64,7 +58,12 @@ func equalRegionStartKey(key, regionStartKey []byte) bool {
 	return bytes.Equal(key, regionStartKey)
 }
 
-func (s *KVStore) splitBatchRegionsReq(bo *Backoffer, keys [][]byte, scatter bool, tableID *int64) (*tikvrpc.Response, error) {
+type SplitResponse struct {
+	splitResp *pdpb.SplitRegionsResponse
+	Error     error
+}
```

Comment:
`SplitResponse` is not necessary I think.

Reply:
Because I want to use a struct{} to receive SplitRegionsResponse and Error and return it to chan